crates/wadm/src/consumers/manager.rs

```diff
@@ -83,6 +83,11 @@ pub trait WorkerCreator {
     ) -> anyhow::Result<Self::Output>;
 }
 
+#[async_trait::async_trait]
+pub trait ConcernedThing {
```

Comment:
Lol this name. Maybe change it to something like `Interest` or `ConcernedWith`

Reply:
Oh boy. This is leftover from some experimentation